it/controllers/benefits/assets/AssetsTransfersBenefitsAmountControllerISpec.scala

```diff
@@ -44,8 +45,6 @@ class AssetsTransfersBenefitsAmountControllerISpec extends IntegrationTest with
   val formLink: String =
     s"/update-and-submit-income-tax-return/employment-income/$taxYearEOY/benefits/assets-to-keep-amount?employmentId=$employmentId"
 
-  private val userRequest = User(mtditid, None, nino, sessionId, affinityGroup)(fakeRequest)
-
   private def employmentUserData(hasPriorBenefits: Boolean, employmentCyaModel: EmploymentCYAModel): EmploymentUserData =
     EmploymentUserData(sessionId, mtditid, nino, taxYearEOY, employmentId, isPriorSubmission = true, hasPriorBenefits = hasPriorBenefits, employmentCyaModel)
```

Comment:
could this method incorporate your new builders?

Reply:
This is a new thing and I have noticed it after a rebase. I will change it now.